browser/ai_chat/common/constants.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
why this file in `brave/browser` and not in `brave/components`?

Reply:
Because constants is only used by modules in brave/browser